src/StreamJsonRpc.Desktop/Resources.resx

```diff
@@ -121,6 +121,14 @@
     <value>Both readable and writable are null.</value>
     <comment>'readable' and 'writable' are parameter names and should not be localized.</comment>
   </data>
+  <data name="ConflictingMethodAttributeValue" xml:space="preserve">
+    <value>The {0} Name value of {1} maps to multiple methods.</value>
```

Comment:
I think this could be more clearly actionable to the user if it were reworded to:

> .NET methods "{type1}.{method1}" and "{type2}.{method2}" cannot both map to the same request method name: {requestMEthodName}."

Mentioning the attribute and property name I think is not necessary. They'll figure it out if we point them directly at the problematic methods.